Added global variable "ol" to eslint config file.

filipheymans opened the pull request.

filipheymans committed:
Added global variable "ol" to eslint config file.

daften reviewed (commented):
@meirege @Jeroen005 @delrueba Is there an option to keep this file without ol, include this in the e.g. maps repo and extend it then? Could we just add a extends in a eslint.rc file in the repo that refers to raw.githubusercontent... ?

filipheymans commented:
I'm closing this PR since its possible to extend this config file.

- Edited .codeclimate.yml and changed fetch path for .eslintrc to ".digipolisgent/.eslintrc"
- Created own ".eslintrc"
`{
  "extends": ".digipolisgent/.eslintrc",
  "root": true,
  "globals": {
    "Drupal": true,
    "drupalSettings": true,
    "jQuery": true,
    "ol": true
  }
}`
- Updated .gitignore: ".eslinrc" to ".digipolisgent/.eslintrc"

daften commented:
@filipheymans Can you document this in the general README file?
